testing/matrix-sdk-integration-testing/assets/ci-start.sh

```diff
@@ -8,6 +8,8 @@ echo " ====== Patching for CI  ====== "
 sed -i 's/^#enable_registration_without_verification:.*$/enable_registration_without_verification: true/g' /data/homeserver.yaml
 sed -i 's/^#enable_registration:.*$/enable_registration: true/g' /data/homeserver.yaml
 echo """
+enable_registration: true
+enable_registration_without_verification: true
 
```

Comment:
if we add them, we don't have to `sed` ...
```suggestion
echo """
enable_registration: true
enable_registration_without_verification: true

```